script/test

```diff
@@ -1,10 +1,21 @@
 #!/bin/bash -e
 
-SERVICES="atst authnid fundz"
-cd $(dirname $0)/../services
+cd $(dirname $0)/..
+source services.ini
+cd services
```

Comment:
This won't work if the directory doesn't exist yet, right? We need to either create the directory, or not git ignore it.